src/lib/encoder.ts

```diff
@@ -325,6 +327,22 @@ const valueTypeEncodingMap = {
     decode: (value: string) => abiCoder.decodeParameter('address', value),
   },
   // NOTE: We could add conditional handling of numeric values here...
+  uint128: {
+    encode: (value: string | number) =>
+      '0x' + abiCoder.encodeParameter('uint128', value).substring(34, 66),
+    decode: (value: string) => {
+      if (!isHex(value))
```

Comment:
Can we put curly brackets for `if` statements, so to keep it consistent with the rest of the code.

Like here:

<img width="879" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/31145285/5a315497-3ec6-45f8-8569-1f7102b97a37">